packages/terra-clinical-result/package.json

```diff
@@ -36,7 +36,8 @@
     "terra-icon": "^3.53.0",
     "terra-mixins": "^1.0.0",
     "terra-theme-context": "^1.0.0",
-    "terra-visually-hidden-text": "^2.36.0"
+    "terra-visually-hidden-text": "^2.36.0",
+    "uuid": "^3.4.0"
```

Comment:
Can we update and lock this to `8.2.0` since that is what's being used across Terra?
```suggestion
    "uuid": "8.2.0"
```

Reply:
Updated here - https://github.com/cerner/terra-clinical/pull/868/commits/f932e12eaad49881c42630811d1619d1000807ed#diff-9084f5bebd8f555fe623f1b8c564a60cd145af644151dd4f498daa620d7fd87fR40. But I see warnings when I run lint. Is that fine?
<img width="1260" alt="Screen Shot 2023-06-16 at 7 45 23 PM" src="https://github.com/cerner/terra-clinical/assets/55503680/60fdb1ef-a885-4905-9264-64f1b5c691a7">